test/cli.js

```diff
@@ -45,4 +45,15 @@ describe('command line usage', function() {
 			done();
 		});
 	});
+	it('--htpasswd', function(done) {
+		exec(cmd,
+			[ "--htpasswd=" + path.join(__dirname, "data/htpasswd-test"),
+				"--no-browser",
+				"--test"
+			], opts, function(error, stdout, stdin) {
+			assert(!error, error);
+			assert(stdout.indexOf("Serving") == 0, "serving string not found");
+			done();
+		});
+	});
```

Comment:
I think this test is not particularly useful test case as it only tests that giving the param does not throw some error. There should probably be a dedicated "htpasswd.js" test file, similar to "mount.js" that creates the server and then tests that it returns the proper HTTP 401 status code and WWW-Authenticate header, and if possible also tests right and wrong passwords.